src/styles/blocks/catalog.scss

```
@@ -0,0 +1,108 @@
+.page {
+  &__main {
+    margin: 0 auto;
+  }
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+
+  max-width: $max-main-width;
+
+  padding: 50px 40px;
+
+  &__card {
+
+    &:not(:nth-child(4n)) {
+      margin-right: 48px;
+    }
+
+    &:nth-child(-n + 4) {
+      margin-bottom: 46px;
+    }
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
```

You created a variables for colors, but not for all 🥺 
```suggestion
  border: 1px solid $c-color;
```